packages/synapse-react-client/src/components/entity/page/CreatedByModifiedBy.tsx

```diff
@@ -103,7 +103,15 @@ export function CreatedByModifiedBy(props: CreatedByModifiedByProps) {
   }
 
   return (
-    <Box sx={{ bgcolor: 'grey.100', py: '10px' }}>
+    <Box
```

Comment:
Looks good, would it be easy to hide the `/` element at small screen sizes? That element was intended to visually separate the content, but now that they are on separate lines it looks unnecessary,